### PR TITLE
when feature is disabled remove buttons related to it

### PR DESCRIPTION
**Pull request by Gloria-kuang**
This PR addresses the following issue:
#384 

When features are unselected, the buttons related to them are removed:
Screen Shot 2022-06-15 at 3 15 33 PM

### CODE REVIEW

File: test/controllers/admin/comfy/users_controller_test.rb [hunk 1]
```diff
@@ -76,6 +76,20 @@ class Comfy::Admin::UsersControllerTest < ActionDispatch::IntegrationTest
     assert_response :success
   end
 
+  test "#edit: show forum button if selected " do
+    sign_in(@user)
+    get admin_users_url(subdomain: @domain)
+    @restarone_subdomain.update(forum_enabled: true)
```
Comment (donrestarone) [inline test/controllers/admin/comfy/users_controller_test.rb:82]: @Gloria-kuang both cases will pass regardless if navigation items are present 
Screen Shot 2022-06-16 at 3 08 15 PM

Comment (donrestarone) [reply test/controllers/admin/comfy/users_controller_test.rb:82]: to see pages in the application visit

``` txt
lvh.me:5250/rails/info/routes

```

File: test/controllers/admin/comfy/users_controller_test.rb [hunk 2]
```diff
@@ -76,6 +76,20 @@ class Comfy::Admin::UsersControllerTest < ActionDispatch::IntegrationTest
...
+    assert_select 'a', {count: 1, text: 'Forum'}
+  end
+
+  test "#edit: show blog button if selected " do
+    sign_in(@user)
+    get admin_users_url(subdomain: @domain)
```
Comment (donrestarone) [inline test/controllers/admin/comfy/users_controller_test.rb:88]: `@domain`  and `@restarone_subdomain` are separate instances.

File: test/controllers/admin/comfy/users_controller_test.rb [hunk 3]
```diff
@@ -76,6 +76,20 @@ class Comfy::Admin::UsersControllerTest < ActionDispatch::IntegrationTest
...
+    @restarone_subdomain.update(blog_enabled: true)
+    assert_select 'a', {count: 1, text: 'Blog'}
+  end
+
   test "denies #edit if not permissioned" do
```
Comment (donrestarone) [inline test/controllers/admin/comfy/users_controller_test.rb:93]:
``` ruby

  test "does not show forum button if not enabled" do
     sign_in(@user)
     get root_url(subdomain: @domain)
     @domain(blog_enabled: true)
     assert_select 'a', {count: 0, text: 'Blog'}
   end
```

File: test/controllers/admin/comfy/users_controller_test.rb [hunk 4]
```diff
@@ -76,6 +76,40 @@ class Comfy::Admin::UsersControllerTest < ActionDispatch::IntegrationTest
...
+    sign_in(@user)
+    get admin_users_url(subdomain: @domain)
+    @restarone_subdomain.update(forum_enabled: true)
+    assert_select 'a', {count: 1, text: 'Forum'}
+  end
+
+  test "#edit: show blog button if selected " do
+    sign_in(@user)
+    get admin_users_url(subdomain: @domain)
+    @restarone_subdomain.update(blog_enabled: true)
```
Comment (donrestarone) [inline test/controllers/admin/comfy/users_controller_test.rb:89]: @Gloria-kuang do we still need this line?

File: test/controllers/admin/comfy/users_controller_test.rb [hunk 5]
```diff
@@ -76,6 +76,40 @@ class Comfy::Admin::UsersControllerTest < ActionDispatch::IntegrationTest
...
+    assert_select 'a', {count: 1, text: 'Blog'}
+  end
+  
+  test "#edit: hide forum button if unselected " do
+    sign_in(@user)
+    get '/mailbox'
```
Comment (donrestarone) [inline test/controllers/admin/comfy/users_controller_test.rb:95]: is this necessary?

``` ruby
get '/mailbox'
```

File: test/controllers/admin/comfy/users_controller_test.rb [hunk 6]
```diff
@@ -76,6 +76,40 @@ class Comfy::Admin::UsersControllerTest < ActionDispatch::IntegrationTest
...
+    payload = {
+      forum_enabled:false
+    }
+    patch web_settings_url(subdomain: @domain, id: @user.id), params: payload
+    assert_select 'a', {count: 0, text:'Forum'}
+  end
+  
+  test "#edit: hide blog button if unselected " do
+    sign_in(@user)
+    get '/mailbox'
```
Comment (donrestarone) [inline test/controllers/admin/comfy/users_controller_test.rb:105]: is this necessary?

``` ruby
get '/mailbox'
```

File: test/controllers/admin/comfy/users_controller_test.rb [hunk 7]
```diff
@@ -76,6 +76,40 @@ class Comfy::Admin::UsersControllerTest < ActionDispatch::IntegrationTest
     assert_response :success
   end
 
+  test "#edit: show forum button if selected " do
+    sign_in(@user)
+    get admin_users_url(subdomain: @domain)
+    @restarone_subdomain.update(forum_enabled: true)
```
Comment (donrestarone) [inline test/controllers/admin/comfy/users_controller_test.rb:82]: @Gloria-kuang is the following line necessary?

``` ruby
@restarone_subdomain.update(forum_enabled: true)
```

**Comment by Gloria-kuang** [comment]
Hi Shashike,

Sure! Let's have a chat tomorrow afternoon whenever you have time！

**Pull request merged**